wdfn-server/waterdata/templates/base_network.html

```diff
@@ -6,7 +6,7 @@
 
 {% block page_script %}
     <script src="{{ 'scripts/vendor.js' | asset_url }}"></script>
-    <script async src="{{ 'network-bundle.js' | asset_url }}"></script>
+    <script src="{{ 'network-bundle.js' | asset_url }}"></script>
```

Comment:
Removed the async. The USWDS javascript should be available prior to the html being loaded so that elements such as the navigation drop downs are not momentarily visible.

Reply:
Thanks Mary, these tips really help illuminate some or the build magic that was going on.